## 06_fix_spelling_error by HIGUCHI Daisuke (VDR dai) <dai@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: eliminate lintian warning: spelling-error-in-{binary,manpages}

--- a/cmd/wtoc/wtoc.c
+++ b/cmd/wtoc/wtoc.c
@@ -195,7 +195,7 @@ main(argc,argv)
 #endif
   }
   else { /* 引数が不正 */
-    fprintf(stderr,gettxt("cannacmd:50", "Usage: wtoc [-f part-of-speach table] [wnndic] [cannadic]\n"));
+    fprintf(stderr,gettxt("cannacmd:50", "Usage: wtoc [-f part-of-speech table] [wnndic] [cannadic]\n"));
     exit(2);
   }
   if( argc >= (2 + option*2) ) { /* Ｗｎｎ辞書をオープン */
--- a/cmd/ctow/ctow.c
+++ b/cmd/ctow/ctow.c
@@ -256,9 +256,9 @@ char *argv[];
       hinshis = chghinshi(h, hinshiSize, taiou, fsize);
       if (!strcmp(hinshis,"")) {
 	fprintf(stderr,gettxt("cannacmd:13", 
-	      "reading:%s nomination:%s a part of speach:%s\n"),y,k,h);
+	      "reading:%s nomination:%s a part of speech:%s\n"),y,k,h);
 	fprintf(stderr,gettxt("cannacmd:14", 
-	      "This part of speach is undefined. Cannot convert.\n"));
+	      "This part of speech is undefined. Cannot convert.\n"));
       }
       else {
 	for ( ; *hinshis; hinshis++, p++) {
--- a/lib/canna/lisp.c
+++ b/lib/canna/lisp.c
@@ -3610,7 +3610,7 @@
         numtostr(xnum(e2), ee);
         num = howManyCharsAre(ss, ee, &sku, &sten, &cs);
         if (num <= 0) {
-          error("Inconsistent range of charcter code ", buf);
+          error("Inconsistent range of character code ", buf);
           /* NOTREACHED */
         }
         kigolen = kigolen + (cswidth[cs] + 1) * num;
--- a/lib/canna/uilib.man
+++ b/lib/canna/uilib.man
@@ -809,7 +809,7 @@ data input:
 	    int length;		        /* length of echo string */
 	    int revPos;                 /* reverse position  */
 	    int revLen;                 /* reverse length    */
-	    unsigned long info;		/* other informations */
+	    unsigned long info;		/* other information */
 	    unsigned char *mode;	/* mode information */
 	    struct {
 		unsigned char *line;	/* a grance of Kanji characters */
--- a/lib/canna/uilib.jmn
+++ b/lib/canna/uilib.jmn
@@ -749,7 +749,7 @@ XCompose構造体へのポインタを指定します。
 	    int revPos;                 /* reverse position  */
 	    int revLen;                 /* reverse length    */
 .mc \(br
-	    unsigned long info;		/* other informations */
+	    unsigned long info;		/* other information */
 	    unsigned char *mode;	/* mode information */
 	    struct {
 		unsigned char *line;	/* a grance of Kanji characters */
